### PR TITLE
Change implementations/EpicGames to implementations/typescript #166

### PR DESCRIPTION
**Pull request by gunsha**
## Relevant components:
- [ ] Signalling server
- [ ] Frontend library
- [x] Frontend UI library
- [ ] Matchmaker
- [ ] Platform scripts
- [ ] SFU

## Problem statement:
What problem does this PR address?
rename implementations/EpicGames to implementations/typescript as mentioned on #166
## Solution
How does this PR solve the problem?

## Documentation
If you added a new feature or changed an existing feature please add some documentation here.

Or better yet, link to the relevant `/Docs` update in your PR.

## Test Plan and Compatibility
What steps have you taken to ensure this PR maintains compataibility with the existing functionality? 

Or better yet, link to the unit tests that accompany this PR.

### CODE REVIEW

**Review by lukehb — CHANGES_REQUESTED** [review]
Seems to be missing Windows setup frontend -
 https://github.com/EpicGames/PixelStreamingInfrastructure/blob/dce89db3f66fc8b545e4d2e6ef2912c17596f2ff/SignallingWebServer/platform_scripts/cmd/setup_frontend.bat#L68

**Comment by Belchy06** [comment]
Are the workflow files per repo or are they modifiable per branch as well?

If they can be modified per branch, they will also need to be updated.

**Comment by lukehb** [comment]
This is an automated reply: I will be out of office for the next two weeks. Please bump this ticket in two weeks.

File: SignallingWebServer/platform_scripts/cmd/setup_frontend.bat [hunk 1]
```diff
@@ -62,10 +62,10 @@
   popd
   echo End of build PS frontend lib step.
 
-  @Rem Do npm install in the Frontend\implementations\EpicGames directory (note we use start because that loads PATH)
+  @Rem Do npm install in the Frontend\implementations/typescript directory (note we use start because that loads PATH)
   echo ----------------------------
   echo Building Epic Games reference frontend...
-  pushd %CD%\Frontend\implementations\EpicGames
+  pushd %CD%\Frontend\implementations/typescript
```
Comment (Belchy06) [inline SignallingWebServer/platform_scripts/cmd/setup_frontend.bat]: This needs to be `\typescript`

**Review by Belchy06 — CHANGES_REQUESTED** [review]

File: SignallingWebServer/platform_scripts/cmd/setup_frontend.bat (hunk 1 above)
Comment (gunsha) [reply SignallingWebServer/platform_scripts/cmd/setup_frontend.bat]: @Belchy06 fixed

**Review by Belchy06 — APPROVED** [review]

**Comment by Belchy06** [comment]
Thanks for fixing those requested changes @gunsha.

We're currently winding down active development of the master branch in preparation for the 5.2 launch, meaning we're only merging critical bug fixes.

We'll open back up to features once 5.2 has launched.

**Comment by Belchy06** [comment]
Rebased this PR on master. Once @lukehb approves it should be good to merge!

**Review by lukehb — APPROVED** [review]

**Pull request merged**